wolfcrypt/src/tfm.c

```diff
@@ -657,7 +658,17 @@ int fp_div(fp_int *a, fp_int *b, fp_int *c, fp_int *d)
 
   fp_init(t1);
   fp_init(t2);
+
+/* Appease compiler for configurations known to need it.
+**   CONFIG_COMPILER_OPTIMIZATION_PERF: Espressif -O2 */
+#if defined(CONFIG_COMPILER_OPTIMIZATION_PERF)
+  fp_init(x); /* Init x to appease some compilers before call to get a copy. */
```

Comment:
I'd prefer to see a PR where these are always initialized.

Reply:
Agreed. See latest code changes.